dev/ci/docker_linux/Dockerfile

```diff
@@ -67,10 +68,12 @@ RUN apt-get update && apt-get install -y --no-install-recommends \
   ruby-dev && \
   apt-get clean
 
-ENV JAVA_HOME="/usr/lib/jvm/adoptopenjdk-8-hotspot-amd64"
+ENV JAVA_HOME="/usr/lib/jvm/adoptopenjdk-11-hotspot-amd64"
+
+RUN export CLASSPATH='/usr/share/jaxb/lib'
```

Comment:
```suggestion
ENV CLASSPATH='/usr/share/jaxb/lib'
```

Reply:
since every `RUN` statement runs in its own shell, I don't think this statement affects any subsequent statements

Reply:
Hmm, you are correct. This leads me to believe that perhaps the jaxb import is not needed with the update to the commandline tools.